src/lti/LTI_OIDC_Login.php

```diff
@@ -112,7 +112,9 @@ protected function validate_oidc_login($request) {
             throw new OIDC_Exception("Could not find login hint", 1);
         }
 
+        // if request['aud'] use it, else try $request['client_id] else set to null;
         $clientId = isset($request['aud']) ? $request['aud'] : null;
+        $clientId = (empty($clientId) && isset($request['client_id'])) ? $request['client_id'] : null;

```

Comment:
This is the functional change. How do we feel about this approach rather than a traditional if block?

Reply:
As discussed, let's flip the priority on these (`client_id` first, then `aud`) and turn it into a conditional.